ops-bedrock/docker-compose-devnet.yml

```diff
@@ -12,10 +12,11 @@ volumes:
 
 services:
   da:
-    platform: linux/x86_64
+    #platform: linux/x86_64
```

Comment:
Why is this commented out? Does it need to be removed?

Reply:
restored.